src/lib.rs

```diff
@@ -724,7 +724,6 @@ impl Peripherals {
     }
     #[doc = r" Unchecked version of `Peripherals::take`"]
     pub unsafe fn steal() -> Self {
-        debug_assert!(!DEVICE_PERIPHERALS);
```

Comment:
Hm this change is pretty silly here, would probably make sense to do this in a separate commit/PR

Reply:
Yes, it's from latest svd2rust.

Reply:
I'll leave it in here then (and make sure it doesn't leak into other PRs)